iam/api/QuickStartNewTest/QuickStartNewTest.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Iam.v1;
+using Google.Apis.Iam.v1.Data;
+using Xunit;
+
+public class QuickStartNewTest : IDisposable
+{
+    private readonly string _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+    private ServiceAccount serviceAccount;
+    private IamService iamService;
+
+    public QuickStartNewTest()
+    {
+        // Create service account for test
+        var credential = GoogleCredential.GetApplicationDefault()
+            .CreateScoped(IamService.Scope.CloudPlatform);
+        iamService = new IamService(
+            new IamService.Initializer
+            {
+                HttpClientInitializer = credential
+            });
+
+        var request = new CreateServiceAccountRequest
+        {
+            AccountId = "iam-test-account",
+            ServiceAccount = new ServiceAccount
+            {
+                DisplayName = "iamTestAccount"
+            }
+        };
+        serviceAccount = iamService.Projects.ServiceAccounts.Create(
+            request, "projects/" + _projectId).Execute();
+    }
+
+    public void Dispose()
+    {
+        // Delete service account
+        string resource = "projects/-/serviceAccounts/" + serviceAccount.Email;

```

Comment:
I think there should be a type ServiceAccountName that would do the parsing abd building of resource names for you. Can you check and change if there's one?

Reply:
I haven't encountered one, but I'm not sure how I would go about looking for it. Is there a specific reference I would look at?